src/test/regress/sql/metadata_sync_from_non_maindb.sql

```diff
@@ -1,8 +1,16 @@
+
```

Comment:
let's delete unnecessary newline additions in the file

Reply:
Removed

Reply:
there are still some in this file